modular_skyrat/modules/medical/code/wounds/synth/robotic_slash.dm

```diff
@@ -470,25 +476,6 @@
 	if (spark)
 		do_sparks(rand(process_shock_spark_count_min, process_shock_spark_count_max), FALSE, victim)
 
-/datum/wound/electrical_damage/proc/can_zap(atom/target = victim, atom/connecting_item, uses_victim_hands, uses_target_hands)
-	if (!isliving(target))
-		return FALSE
-	if (target == victim)
-		return TRUE
-
-	if (!isnull(connecting_item))
-		return FALSE
-	if (HAS_TRAIT(target, TRAIT_SHOCKIMMUNE))
-		return FALSE
-	if (uses_victim_hands && victim.wearing_shock_proof_gloves())
-		return FALSE
-	if (uses_target_hands && iscarbon(target))
-		var/mob/living/carbon/carbon_target = target
-		if (carbon_target.wearing_shock_proof_gloves())
-			return FALSE
-
-	return TRUE
```

Comment:
I couldn't tell, was this moved somewhere else, or just completely removed?

Reply:
removed as it was never used